utils/git.go

```diff
@@ -267,3 +279,40 @@ func (m *manager) readRevisionFromPackedRef(ref string) {
 	m.err = errorutils.CheckError(errors.New("failed fetching revision from git config, from ref: " + ref))
 	return
 }
+
+func (m *manager) readMessage() {
+	if m.err != nil {
+		return
+	}
+	path := m.getPathHandleSubmodule()
+	r, err := git.PlainOpenWithOptions(path, &git.PlainOpenOptions{DetectDotGit: false})
```

Comment:
Please change "r" to something more informative, for example:
```suggestion
	gitRepo, err := git.PlainOpenWithOptions(path, &git.PlainOpenOptions{DetectDotGit: false})
```